### DISCUSSION TITLE
simulate the RLC buffer

### DISCUSSION THREAD

**Discussion by algeady** [Unipisa/Simu5G#241] [Q&A] [answered]
Hallo can anyone please help me how to simulate the RLC buffer in the gnb  to make a congestion Simulation am a beginner thanks a lot

**Comment by giovanninardini** [comment, marked as answer]
The RLC buffer is already modeled within Simu5G. You can simulate congestion by increasing the number of UEs served by the gNB and/or increasing the traffic sent by UEs.
